spec/integration/recipes/notifies_spec.rb

```diff
@@ -135,8 +140,9 @@
           resource_name :notifying_test
 
           action :run do
-            log "bar" do
+            notify_group "bar" do
               notifies :write, 'log[foo]', :delayed
+              action :run
```

Comment:
If the default is to only use these as a notifier, why are you setting an action on them?

Reply:
`action :nothing` doesn't fire notifies, and that's hard coded in Chef::Resource and difficult to override.

Reply:
eh, no, that was before coffee.

in those specs i'm abusing it deliberately just to test notifications, that isn't an example of how to actually use it.

Reply:
got it.